Assignment2/AFGranado-AnlleloGranado.ttl

```diff
@@ -0,0 +1,15 @@
+@base <http://www.fi.upm.es/ontologies/>
+@prefix vocab : <http://www.fi.upm.es/ontologies#>
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+
+:Class01	vocab:includes :Sensor029;
+		vocab:includes :Computer101;
+
+:Sensor029	vocab:hasMeasurement :Measurement8401;
+
+:Measurement8401	vocab:hasTemperature "29"^^xsd:int;
+			vocab:atTime "2010-06-12T12:00:12"^^xsd:dateTime.
+
+:Computer101	vocab:hasOwner :User10A.
+
+:User10A	vocab:hasName :"Pedro"^^xsd:string.
```

Comment:
This is still wrong. No : symbol before the string "Pedro", please.

Reply:
reviewed and it is correct now